internal/exec/util/user_lookup.go

```diff
@@ -48,3 +48,26 @@ func (u Util) userLookup(name string) (*user.User, error) {
 
 	return usr, nil
 }
+
+// groupLookup looks up the group in u.DestDir.
+func (u Util) groupLookup(name string) (*user.Group, error) {
```

Comment:
You sure? Looks updated on my end.

Just pushed another commit, maybe that'll poke GH into seeing the change.

![2017-05-15_14 48 12_3200x1800_scrot](https://cloud.githubusercontent.com/assets/2439413/26080968/9aff686c-397d-11e7-9946-2c7ad575f01b.png)